Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -476,12 +495,137 @@ void MemoryWidget::OnSearchAddress()
   m_search_offset->setPalette(offset_palette);
 }
 
-void MemoryWidget::ValidateSearchValue()
+void MemoryWidget::ValidateAndPreviewInputValue()
 {
+  m_data_preview->clear();
+  QString input_text = m_data_edit->text();
+
+  if (input_text.isEmpty())
+    return;
+
+  // Remove any spaces
+  if (!m_input_ascii->isChecked())
+    input_text = input_text.remove(QLatin1Char(' '));
```

Comment:
```suggestion
    input_text.remove(QLatin1Char(' '));
```

Since `remove()` modifies the string in-place, so you don't need to copy it

Reply:
lol the simplest thing and I keep messing it up. Thanks again.